src/hdf5/HDF5IO.cpp

```diff
@@ -9,7 +9,7 @@
 #include <H5Cpp.h>
 #include <H5Fpublic.h>
 
-#include "Utils.hpp"
+#include "../Utils.hpp"
```

Comment:
What is the motivation for switching to relative paths on the includes?

Reply:
When I was including the aqnwb source code as a subfolder in OpenEphys `nwb-format/Source` subfolder and compiling it all together, I  would get errors that the header files could not be found.

However I realized since then that another option is to simply add `include_directories(${SOURCE_PATH}/aqnwb)` to the `nwb-format/CMakeLists.txt` file. We can just take that approach instead if it's more readable.

Reply:
> However I realized since then that another option is to simply add `include_directories(${SOURCE_PATH}/aqnwb)` to the `nwb-format/CMakeLists.txt` file. We can just take that approach instead if it's more readable

That would be my personal preference